build-system/tasks/integration.js

```diff
@@ -40,11 +44,34 @@ class Runner extends RuntimeTestRunner {
   }
 }
 
+async function buildTransformedHtml() {
+  const filePaths = await globby('./test/fixtures/**/*.html');
+  let normalizedFilePath;
+  try {
+    log(
+      green('Copying'),
+      cyan('Integration files'),
+      green('to'),
+      cyan(`test-bin/`) + green('...')
+    );
```

Comment:
nit: Since the stuff in `cyan` is just part of the sentence, maybe rewrite as follows?

```suggestion
    log(
      green('Copying integration test files to'),
      cyan(`test-bin/`) + green('...')
    );
```

Reply:
Would also recommend just using a standard string for `test-bin/`.